### PR TITLE
Dockerfile: optional move to alpine

### PR DESCRIPTION
**Pull request by bryanlatten**
Will be trigger via Docker Hub's alternate Dockerfile build system, while remaining in master!

Appears in Docker Hub as behance/docker-nginx:5.0-alpine

### CODE REVIEW

File: Dockerfile-alpine [hunk 1]
```diff
@@ -0,0 +1,48 @@
+FROM alpine:3.4
```
Comment (bryanlatten) [inline Dockerfile-alpine]: note, this is called `Dockerfile-alpine` to be configured in Docker Hub to build separately

File: container/root/etc/nginx/nginx.conf [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # at build time.
 #
 # For run-time replacements, ie, consuming environment vars,
-# add to the run.d/nginx script
+# add to the /etc/cont-init.d/nginx script
```
Comment (bryanlatten) [inline container/root/etc/nginx/nginx.conf]: had never been updated from the S6 conversion...

File: container/root/etc/nginx/sites-available/default [hunk 3]
```diff
@@ -1,6 +1,8 @@
 server {
   listen 8080;
 
+  root /var/www/html;
```
Comment (bryanlatten) [inline container/root/etc/nginx/sites-available/default]: always specify. ubuntu had different layout than alpine

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+web:
```
Comment (bryanlatten) [inline docker-compose.yml]: for quick and easy testing

**Comment by ko-be** [comment]
👍

**Comment by Attamusc** [comment]
👍

**Comment by mannytoledo** [comment]
👍

**Comment by bossjones** [comment]
👍

**Pull request merged**